presto-native-execution/presto_cpp/main/http/HttpClient.cpp

```diff
@@ -138,6 +144,7 @@ class ResponseHandler : public proxygen::HTTPTransactionHandler {
  private:
   const proxygen::HTTPMessage request_;
   const std::string body_;
+  velox::memory::MemoryPool* const pool_;
```

Comment:
Should we enforce pool_ to be non-null?

Reply:
Yeah, we need to add a default memory pool in Announcement then.